Doc/library/traceback.rst

```diff
@@ -42,6 +42,12 @@ The module defines the following functions:
    :term:`file <file object>` or :term:`file-like object` to
    receive the output.
 
+   Note that the meaning of the *limit* parameter is different than the meaning
+   of :const:`sys.tracebacklimit`. Negative *limit* values correspond to
+   positive values of  :const:`sys.tracebacklimit`, whereas the behaviour of
+   positive values of *limit* cannot be achieved with
+   :const:`sys.tracebacklimit`.
```

Comment:
```suggestion
   Note that the meaning of the *limit* parameter is different than the meaning
   of :const:`sys.tracebacklimit`. Negative *limit* values correspond to
   positive values of :const:`sys.tracebacklimit`, whereas the behaviour of
   positive values of *limit* cannot be achieved with
   :const:`sys.tracebacklimit`.
```

Reply:
(I also flipped the `positive values of limit` to `positive limit values` since you used `negative limit values` first).